mathics/core/evaluation.py

```diff
@@ -140,7 +141,7 @@ def run_with_timeout_and_stack(request, timeout, evaluation):
     if success:
         return result
     else:
-        raise result[0].with_traceback(result[1], result[2])
+        raise result[1].with_traceback(result[1], result[2])
```

Comment:
There single parameter to `with_traceback()` should be a tb parameter: result[2]. See PR.

Reply:
OK. I just copied the line 143 from `mathics.core.evaluation`. Probably we need to fix it there too.

Reply:
Both places should be fixed in the updated PR.